tests/test_setup_bind_mounts

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
+
+. "${srcdir:-.}/common.sh"
+
+cat >"$TMP/snap.name.app" <<EOF
+@unrestricted
+EOF
+
+cat >"$TMP/snap.name.fstab" <<EOF
+/src /dst none bind 0 0
+EOF
+
+printf "Test that bind mounts work "
+"$L" snap.name.app /bin/true
```

Comment:
It would be better to test that they *really* work by creating /src/foo and checking it exists in /dst/foo